datanode/api/errors.go

```diff
@@ -214,7 +214,7 @@ var (
 	ErrCopyHistorySegmentToFile            = errors.New("failed to copy history segment to file")
 	ErrGetIpfsAddress                      = errors.New("failed to get node's ipfs address")
 	ErrNetworkHistoryNoTableName           = errors.New("no table name for network history supplied")
-	ErrNetworkHistoryGetContiguousSegments = errors.New("failed to get contiguous history segments")
+	ErrNetworkHistoryGetContiguousSegments = newInvalidArgumentError("failed to get contiguous history segments")
```

Comment:
Minor: can we just change the text from `failed` to `could not` just so that it sounds less like its core's fault?